app/src/main/java/org/fossasia/openevent/general/event/topic/SimilarEventsFragment.kt

```diff
@@ -153,4 +168,31 @@ class SimilarEventsFragment : Fragment() {
         moreLikeThis.isGone = similarEvents.isEmpty()
         similarEventsRecycler.isGone = similarEvents.isEmpty()
     }
+
+    private fun setUpAdapter() {
+        similarEvents.clear()
+        var id: Long
+        var add: Boolean
+        if (similarIdEvents.size != 0 && similarLocationEvents.size != 0) {
+            similarIdEvents.forEach {
+                id = it.id
+                add = true
+                similarLocationEvents.forEach {
+                    if (id == it.id) add = false
```

Comment:
You can add a `break` statement to prevent iterating after `add` has already been set to `false`

Reply:
yes, thanks didn't notice that.

Reply:
resolved